test/Runtime/fatal-message-crash-dump-win.swift

```diff
@@ -0,0 +1,43 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift %s -o %t/a.out
+// RUN: %target-codesign %t/a.out
+// RUN: not %target-run %t/a.out 2>&1 | %FileCheck %s
+// REQUIRES: executable_test
+// REQUIRES: OS=windows-msvc
+
+// CHECK: Fatal error message: a/fatal-message-crash-dump-win.swift:{{[0-9]+}}: Fatal error: Fatal crash!
+
+import WinSDK
+
+func crash() {
+    fatalError("Fatal crash!")
+}
+
+func setupHandler() {
+    SetUnhandledExceptionFilter { exception_pointers in
+        guard let hSwiftCore = GetModuleHandleA("swiftCore.dll") else {
+            return EXCEPTION_EXECUTE_HANDLER
+        }
+        let ppLastFatalErrorMessage = unsafeBitCast(
+            GetProcAddress(hSwiftCore, "gLastFatalErrorMessage"),
+            to: UnsafeMutablePointer<UnsafeMutablePointer<UInt8>>.self)
```

Comment:
Is it possible to cast it to `Optional<UnsafeMutablePointer<Optional<UnsafeMutablePointer<UInt8>>>>` instead? That way you can compare it for nil using optional instead of the bit pattern.

Reply:
Done, PTAL